strapi/strapi
# fix: application crash when deleting a filter on a contenttype

### What does it do?

Bind `onClick` prop for all usages of `Tag` component instead of their wrapper.

### Why is it needed?

`Tag` component calls `onClick` without checking `onClick` is bind or not. It causes app crashes when we attempt to delete a filter on a content type.

https://github.com/strapi/design-system/blob/5eb7cfd5a44545c8243a6c15706f066e15918a73/packages/strapi-design-system/src/Tag/Tag.js#L25-L46

(we may need to update Tag implementation too)

### How to test it?

-  Go to the content manager page and select a content type.
- Add a filter on the table list
- Attempt to remove this.
- Notice the error message displayed on the console.


### Demo

https://user-images.githubusercontent.com/45232708/205858001-528863b1-e8f0-45d1-9b99-b699a74e3ac8.mov


### Related issue(s)/PR(s)

- Fixes https://github.com/strapi/strapi/issues/14470
